homeassistant/components/tuya/config_flow.py

```diff
@@ -121,12 +120,16 @@ async def async_step_login(self, user_input=None):
             )
 
             if response.get("success", False):
-                _LOGGER.debug("TuyaConfigFlow.async_step_user login success")
+                _LOGGER.debug(
+                    "TuyaConfigFlow.async_step_user login success - %s",
+                    response,
+                )
                 return self.async_create_entry(
                     title=user_input[CONF_USERNAME],
                     data=user_input,
                 )
             errors["base"] = RESULT_AUTH_FAILED
+            _LOGGER.error("TuyaConfigFlow.async_step_user login failed - %s", response)
```

Comment:
We know the module from the logger already. I suggest something like:
```suggestion
            _LOGGER.error("Login failed: %s", response)
```

Reply:
Correct, but this is in line with the rest of the file.
IMO all of them should be changed but did not want to do that in this PR.

Reply:
All of them are touched in this PR :smile: (I'll update it :+1: )